x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form.tsx

```diff
@@ -76,6 +81,12 @@ export function SloEditForm({ slo }: Props) {
     setIsAddRuleFlyoutOpen(true);
   }
 
+  useEffect(() => {
+    if (isEditMode && rules && rules[slo.id].length && isCreateRuleCheckboxChecked) {
```

Comment:
What do you think about unchecking this box in edit mode regardless of having a rule or not? If there was a reason for the user to uncheck it in the first place, maybe we could keep it that way.

Reply:
@katrin-freihofner @grabowskit 
What do you think about unchecking the rule creation box in edit mode regardless of having a rule or not? If there was a reason for the user to uncheck it in the first place, maybe we could keep it that way.